docs/Crashes.md

```diff
@@ -0,0 +1,44 @@
+# Crashes
+
+When running on Windows, Chatterino 2.4.1 and later will automatically save information about a crash in a [minidump](https://docs.sentry.io/platforms/native/guides/minidumps/).
+
+Crashdumps are saved inside the `Crashes/reports` folder in your [Chatterino folder](Settings/#where-is-my-chatterino-folder-located).
```

Comment:
```suggestion
Crashdumps are saved inside the `Crashes/reports` folder in your [Chatterino folder](/Settings/#where-is-my-chatterino-folder-located).
```

Another alternative is to use `../` - I'm fine with either (we haven't cross-linked before)